artifactory/services/upload.go

```diff
@@ -993,6 +993,10 @@ func (us *UploadService) addFileToZip(artifact *clientutils.Artifact, progressPr
 	if errorutils.CheckError(err) != nil {
 		return
 	}
+	// Increment general progress by 1 for each file added to the zip.
+	if us.Progress != nil {
```

Comment:
`CreateUploadAsZipFunc` calls `addFileToZip` twice. Wouldn't this increment the progress twice? Or is that intentional?

Reply:
check now @yahavi